hdtv/rootext/fit/ExpBg.cc

```diff
@@ -26,20 +26,18 @@
 
 #include <iostream>
 
+#include <ROOT/RMakeUnique.hxx>
 #include <TError.h>
 #include <TF1.h>
 #include <TH1.h>
 #include <TVirtualFitter.h>
-#include <ROOT/RMakeUnique.hxx>
 
 #include "Util.hh"
 
 namespace HDTV {
 namespace Fit {
 
-ExpBg::ExpBg(int nParams,
-             const Option<bool> integrate,
-             const Option<std::string> likelihood) {
+ExpBg::ExpBg(int nParams, const Option<bool> integrate, const Option<std::string> likelihood) {
```

Comment:
Codacy found an issue: [Function parameter 'likelihood' should be passed by const reference.](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)